src/modules/geolocation/hooks/useGeolocation.tsx

```diff
@@ -1,16 +1,23 @@
 import { useCallback, useState } from 'react';
```

Comment:
* Move hook: `src/modules/geolocation/hooks` => `src/hooks`
* Move input component: `src/modules/geolocation/components` => `src/components/elements/input` as well as related components (Help Dialog, Error Alert)
* Move map components: `src/modules/geolocation/components` => `src/components/elements/maps`
  * Additional generic map components from https://github.com/greenriver/hmis-frontend/pull/997 (excluding EnrollmentLocationMap) will also go there
* Remove `modules/geolocation` since it's not domain-specific to anything

Does this sound right base on our conversation @ttoomey @martha?

Reply:
Updated accordingly. I moved the types file to a global location `src/types/geolocationTypes.ts`.